src/themes/proficiency.ts

```diff
@@ -73,6 +73,7 @@ const metabase: MetabaseTheme = {
       padding: "6px 16px",
     },
     dashboard: {
+      backgroundColor: "transparent",
```

Comment:
Makes the dashboard background transparent, so the gradient backdrop can show up.